app/views/admin/admin_user_edit.php

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
```

Comment:
@tony-tang-aligent  I think there might be some value in having a `header` and `footer` view template so that we're not duplicating these bits of code across numerous templates

Reply:
Brilliant point.